packages/core/test/invoke.test.ts

```diff
@@ -1735,6 +1735,7 @@ describe('invoke', () => {
         JSON.stringify(waitingState);
       }).not.toThrow();
 
+      // TODO: the `src` is resolved now to ObservableActorRef, figure out what's the proper behavior
```

Comment:
The `src` should be a description of the source of the actor ref, not the actual actor ref